pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeFunctions.java

```diff
@@ -261,6 +262,15 @@ public static long now() {
     return System.currentTimeMillis();
   }
 
+  /**
+   * Return time as epoch millis before the given period (represented by as an ISO8601 formatted String).
```

Comment:
Put some example period string in the javadoc?

From the javadoc of `Duration.parse()`:
```
The formats accepted are based on the ISO-8601 duration format {@code PnDTnHnMn.nS} with days considered to be exactly 24 hours.
```